app/controllers/queue_items_controller.rb

```diff
@@ -0,0 +1,35 @@
+class QueueItemsController < ApplicationController
+  before_filter :require_user
+
+  def index
+    @queue_items = current_user.queue_items
+  end
+
+  def create
+    video = Video.find(params[:video_id]) # should be Video.find(queue_items_params)
```

Comment:
The comment is not relevant anymore - after we discussed it , `find` method escapes params safely.